R/release.R

```diff
@@ -72,6 +72,9 @@ release_checklist <- function(version, on_cran) {
   has_pkgdown <- uses_pkgdown()
   has_readme <- file_exists(proj_path("README.Rmd"))
 
+  repo <- github_url_from_git_remotes()
+  has_github_links <- repo %in% desc::desc_get_urls() && glue::glue("{repo}/issues") %in% desc::desc_get("BugReports")
```

Comment:
I'd extract this out into a `has_github_links()` function that lives in `github.R`, but otherwise the approach looks good.